_im-plugin/ism/policies.md

```diff
@@ -206,6 +206,7 @@ Set `index.plugins.index_state_management.rollover_alias` as the alias to rollov
 Parameter | Description | Type | Example | Required
 :--- | :--- |:--- |:--- |
 `min_size` | The minimum size of the total primary shard storage (not counting replicas) required to roll over the index. For example, if you set `min_size` to 100 GiB and your index has 5 primary shards and 5 replica shards of 20 GiB each, the total size of all primary shards is 100 GiB, so the rollover occurs. ISM doesn't check indexes continually, so it doesn't roll over indexes at exactly 100 GiB. Instead, if an index is continuously growing, ISM might check it at 99 GiB, not perform the rollover, check again when the shards reach 105 GiB, and then perform the operation. | `string` | `20gb` or `5mb` | No
+`min_primary_shard_size` | The minimum size of a *single primary shard* storage (not counting replicas) required to roll over the index. For example, if you set `min_primary_shard_size` to 30 GiB and there is a *single primary shard* that has a size greater than the condition, the rollover occurs. ISM doesn't check indexes continually, so it doesn't roll over indexes at exactly 30 GiB. Instead, if an index shard is continuously growing, ISM might check it at 29 GiB, not perform the rollover, check again when the shards reach 32 GiB, and then perform the operation. | `string` | `20gb` or `5mb` | No
```

Comment:
This looks like it should be "The minimum size of the" (instead of "a"). If this is the first appearance of "ISM" on the page, please define as "Index State Management (ISM)". Confirm the use of "continuously" and "continually". Continuously describes an action that happens without ceasing. Continually describes an action that recurs frequently or regularly.

Reply:
Writing "**the** single primary shard" would imply that the shard was **the shard** as in the single one or the main one, but it is not.

The meaning of `The minimum size of _a_ **single primary shard**` is to be interpreted as "any one of multiple shards in an index", as stated in the referenced issue from the index-management repo. The `min_primary_shard_size` parameter applies to "**one** of the primary shards", 

Regarding ISM the use of ISM is consistent through the document, and the document's parent is [Index State Management](https://github.com/opensearch-project/documentation-website/blob/main/_im-plugin/ism/policies.md?plain=1#L5)

The use of "continuously" and "continually" was interesting, I hope I never cease to learn. I replicated what was used in `min_size` from earlier, but I agree that this should be clarified.
In addition, having these long explanations for each condition is not necessary, makes the doc very long winded / wall-of-text-like, and really applies to _all_ conditions.

I'll add a new commit where I move the detailed explanation out of the "pr condition", and refer to it, and hopefully with a clear an consistent language.